Makefile

```diff
@@ -27,11 +27,11 @@ test: test-parallel
 
 ## Lint
 lint:
-	flake8 src tests
-	isort --check-only --diff src tests
-	pydocstyle src tests
-	black --check src tests
-	mypy src tests
+	flake8 src tests || exit 1
+	isort --check-only --diff src tests || exit 1
+	pydocstyle src tests || exit 1
+	black --check src tests || exit 1
+	mypy src tests || exit 1
```

Comment:
was this non blocking before?

Reply:
correct, so we had a bunch of linter errors. I had noticed a few times before when we got contributions from outside the company, but didn't actually look into it until i noticed my own run of `black` producing changes in random other files right now